dashboard/app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -2,8 +2,6 @@
 
 <p>You can confirm your account email through the link below:</p>
 
-<p><%= link_to 'Confirm my account', confirmation_url(@resource, :confirmation_token => @token) %></p>
-
```

Comment:
should we remove this view altogether? I'm assuming it's no longer used at all, but if it is unintentionally referenced somewhere, i think we'd prefer to report an error than send a user an email missing a link.